README.rst

```diff
@@ -31,12 +38,15 @@ Testing Publication
 -------------------
 
 `poetry build`
-`poetry config.repositories.testpypi https://test.pypi.org`
-`poetry publish --repository testpypi`
+
+`poetry config repositories.testpypi https://test.pypi.org/legacy/`
+
+`poetry publish --repository testpypi --username __token__ --password <token_value>`
```

Comment:
Since this is a public repo, I will create a separate README in a different place (probably the mono repo) that has information about where we can find the token and link the file to this repository.